### ISSUE TITLE
documentation

### ISSUE THREAD

**Issue by phtephanx** [jpuigcerver/pytorch-baidu-ctc#3]
Thanks for the bindings!

I simply wanted to point out that you could add to the documentation:
* y, xs and ys need to be on the cpu
* x represents non-logits (no log-softmax applied)

**Comment by WenmuZhou** [comment]
can I use this with the same input as torch.nn.CTCloss?

**Comment by jpuigcerver** [comment]
Hi @WenmuZhou,

Not exactly. You need to make sure that you place the tensors in the appropriate devices. 

Take a look at this piece of code, were I use both implementations of the CTC loss:
https://github.com/jpuigcerver/PyLaia/blob/41d2cc41d742e7ab336393fde8f56585ff49ee52/laia/losses/ctc_loss.py#L350

**Comment by WenmuZhou** [comment]
tks@jpuigcerver

**Comment by WeihongM** [comment]
Hi,  @jpuigcerver   ask for one question: the input pred needs the log_softmax output?
